### PR TITLE
[dev] AB#42504 - PropType Warning Fixes and Added `otherDataGroups` To PersonPanelDetails

### PR DESCRIPTION
**Pull request by cbrewer1689**

### CODE REVIEW

**Comment by groberts314** [comment]
Validation build failure was an issue with `node-gyp` (_that guy!_ ;-) during `npm install` for the UI Docs project.
https://dev.azure.com/saddlebackchurch/Church%20Management/_build/results?buildId=26830&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=295ef4c3-3a46-5f11-020b-b8098f9465c0&s=d654deb9-056d-50a2-1717-90c08683d50a

I queued up a retry to see if it was a transient error or something more serious.

File: src/inputs/select/select.jsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ const propTypes = {
     tabIndex: PropTypes.number,
     underline: PropTypes.bool,
     value: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.shape({})),
```
Comment (groberts314) [inline src/inputs/select/select.jsx:45]: Is this an array of shapes when we're in multi-select mode?

**Review by groberts314 — APPROVED** [review]
LGTM

File: src/inputs/select/select.jsx (hunk 1 above)
Comment (cbrewer1689) [reply src/inputs/select/select.jsx:45]: Yes sir.

**Comment by groberts314** [comment]
@cbrewer1689 I have a possible fix for our `node-gyp` woes in #272.  Check it out and see if that will work.

**Comment by groberts314** [comment]
🎉 Passing now!  :shipit: ...???

**Pull request merged**